### PR TITLE
Add article about sparkpickle

### PR DESCRIPTION
**Pull request by vmarkovtsev**

### CODE REVIEW

File: content/post/reading_pyspark_pickles_locally.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+It is the stream of key-value pairs, where keys and values must have the uniform
+types, and their class names are stored in the stream header. So basically,
+PySpark writes SequenceFile-s with null keys and blob values.
+
+If you want to read those files on your personal computer and not in the Spark
+cluster environment, you are in trouble: there is simply no way of doing it. You
+have to install Spark, PySpark, connect it to your storage backend, process
+the files and finally pickle the result of `collect()`.
+That's why I had to write [src-d/sparkpickle](https://github.com/src-d/sparkpickle):
+it is a tiny Python package with *any* dependencies which allows you to load
```
Comment (Serabe) [inline content/post/reading_pyspark_pickles_locally.md]: s/with/without ?

**Comment by vmarkovtsev** [comment]
@Serabe Thanks! fixed

**Comment by mcuadros** [comment]
Maybe adding a conclusion to the article could be useful

**Comment by vmarkovtsev** [comment]
@mcuadros Added the summary.

**Comment by vmarkovtsev** [comment]
@mcuadros merging this to be on time with Hackerrank

**Pull request merged**